rdir/rdir.c

```diff
@@ -1556,6 +1556,9 @@ grid_main_get_options(void)
 		{"Config", OT_LIST, {.lst = &config_paths},
 			"Load the given file and overload the central variables"},
 
+		{"Servicing", OT_BOOL, {.b = &servicing},
+		        "If set, xattrs lock will be unset on volume"},
```

Comment:
This is wrong. Extended attributes won't be removed. If they exist, they will be checked, if they don't, an error will be returned.